### DISCUSSION TITLE
Request for access to ROSIE software.

### DISCUSSION THREAD

**Discussion by Metotey** [RosettaCommons/rosetta#70] [ROSIE]
Hello,

I am currently pursuing my master’s degree at Mohammed VI University of Health Sciences. I'm looking for access ROSIE in order to carry out analyses that are part of my thesis work.

I would like to clarify that the “@um6ss” included in my email address is an abbreviation for “Université Mohammed VI des Sciences de la Santé,” reflecting my affiliation with the university.

Thank you for considering my request.

**Comment by lyskov** [comment]
Thank you for letting us know! This should be fixed now. Please feel free to re-open if you still having issues.

**Comment by wertick01** [comment]
Hello, I am getting a higher education at the Faculty of Natural Sciences of NSU and I am looking for access to the ROSE software for modeling structures for my thesis. Domain address "@g.nsu.ru ".

**Reply by roccomoretti** [reply]
Unfortunately, due to the current situation there are legal/administrative issues with using servers in the United States to provide computational services to people in Russia, which means that we are unable to approve Russian email addresses for ROSIE accounts at this time.

We apologize for the current state of the world.
